src/opm/parser/eclipse/share/keywords/900_OPM/S/SPOLYMW

```diff
@@ -0,0 +1,8 @@
+{
+  "name" : "SPOLYMW" ,
+  "sections" : ["SOLUTION", "SPECIAL"],
```

Comment:
Please remove the `SPECIAL` - that is a mistake which suddenly happened one day ;-)

Reply:
Yes. You are right.